### DISCUSSION TITLE
Fan-freakin'-tastic work guys!

### DISCUSSION THREAD

**Discussion by NerdyDeedsLLC** [MonitorControl/MonitorControl#1101] [General]
Just wanted to pop in and say: y'all NAILED this one. Works flawlessly on all 4 of my macs, does precisely what it's supposed to (despite having anywhere from 1-4 screens on each, including one routing through a bloody DOCKING STATION to a KVM to adjust a TARGET DISPLAY MODE iMac!? Dayum!)

Outstanding job guys. F'real. I'm thrilled to death to see someone put this much careful thought and consideration into a tool, and the fact that I've yet to see it fail me (given some of the REALLY twisted configurations I have set up on my network, few will really  begin to grasp how much of an accomplishment this is).

Well done, ladies and gentlemen. And I'm saying this as a former Apple developer myself. If you slap this puppy into Homebrew  it's going straight into my DotFiles. Bravo. <3

**Comment by waydabber** [comment]
Thank you! :)
